lib/scooter_service.dart

```diff
@@ -341,6 +351,17 @@ class ScooterService {
     }
   }
 
+  void _updateScooterState() {
+    log("### Internal state: '${_state}' | '${_internalState}'");
+    if (_state != null && _internalState == "hibernating") {
```

Comment:
`_internalState` is always empty string

Reply:
I identified that it's updated less regular. Therefore saving it in the shared prefs as workaround. Eventually this needs to be optimized further...